src/Ledger/Ratify.lagda

```diff
@@ -29,8 +29,10 @@ instance
 \begin{figure*}[h]
 \begin{code}
 record StakeDistrs : Set where
-  field poolStakeDistr  : Credential ↛ Coin
-        drepStakeDistr  : Credential ↛ Coin
+  field drepStakeDistr    : Credential ↛ Coin
```

Comment:
We should probably change `drepStakeDistr` to `VDeleg ↛ Coin`. We'll compute that by combining the `voteDelegs` with the per credential delegation, and that's the type it'll give us. That also means we don't need the `autoVoterStakeDist` special casing.